widget/view.controller.js

```diff
@@ -345,6 +457,21 @@
             'operator': 'count'
           }]
       };
+      if ($scope.config.recordTags) {
+        var playbookExcludeTag = []
+        for (var i = 0; i < $scope.config.recordTags.length; i++) {
+          var tagArr = $scope.config.recordTags[i].split('/');
```

Comment:
'/' will backslash be present in all recordTags or else a check can be applied

Reply:
It has already been handled, if the string contains no '/' later pop is used which returns the last value in the array